docs_src/query_param_models/tutorial001.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from fastapi import FastAPI, Query
+from pydantic import BaseModel, Field
+from typing_extensions import Literal
+
+app = FastAPI()
+
+
+class FilterParams(BaseModel):
+    limit: int = Field(100, gt=0, le=100)
+    offset: int = Field(0, ge=0)
+    order_by: Literal["created_at", "updated_at"] = "created_at"
+    tags: List[str] = []
```

Comment:
It's generally not a good idea to use mutables as defaults.

```suggestion
    tags: List[str] = Field(default_factory=list)
```

Reply:
Thanks! Pydantic does a deep copy of default mutables, so in this case it's not a problem.

Reply:
Oh, good to know 💡